SQLitePCL.pretty/ThrowHelper.cs

```diff
@@ -0,0 +1,125 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+
+namespace SQLitePCL.pretty
+{
+    /// <summary>
+    /// Provides methods for throwing common exceptions.
```

Comment:
it should have an explanation on _why_ it's needed

Reply:
I’m on my phone, so I can’t edit it, but here is the explanation: https://reubenbond.github.io/posts/dotnet-perf-tuning

Reply:
Can you add it when you're back at laptop?